jekyll/_cci2/next-gen-migration-guide.md

```diff
@@ -0,0 +1,73 @@
+
+
+| Legacy Image | Next-Gen Image |
+| --- | --- |
+| circleci/buildpack-deps | cimg/base |
+| circleci/jruby | No suggested path |
+| circleci/dynamodb | No suggested path |
+| circleci/golang | cimg/go |
+
+
+### Browser Testing
+
+With legacy images, there were 4 different variant tags you could use to get browser testing for a particular image. For example, if you were doing browser testing with the Python v3.7.0 image, you might have used Docker image: circleci/python:3.7.0-browsers. These 4 tags have now been consolidated into a single tag designed to be used together with the CircleCI Browser Tools orb.
+
+
+Legacy Variant Tags
+
+
+Next-gen Variant Tag
+-browsers
+-browsers + Browser Tools Orb
+-browsers-legacy
+-node-browsers
+-node-browsers-legacy
+
+
+The new, single browsers variant tag includes Node.js, common utilities to use in browser testing such as Selenium, but not the actual browsers. Please note the browsers are no longer pre-installed. Instead, browsers such as Google Chrome and Firefox as well as their drivers Chromedriver and Gecko are installed via the browsers-tools orb. This provides the flexibility to mix and match the browser versions you need in a build rather than using strictly what CircleCI provides. You can find examples of how to use this orb here.
+
+Using Ubuntu as the Single Base OS
+
+Legacy images had variant tags for several different base operating systems (OS). Some images were versions of Debian and Ubuntu while other images provided several different bases. This is no longer the case. All CircleCI next-gen images are based on the latest LTS release of Ubuntu.
+
+With the base image, at least two LTS releases and non-EOL’d standard releases will be supported.
+
+
+## Troubleshooting
+
+When migrating to a next-gen image, there might be some software issues. Common issues include a library you were using now has a different version or an apt package is no longer pre-installed. In this scenario simply install that package using:
+
+sudo apt-get update && sudo apt-get install -y <the-package>
+
+Every image has its own GitHub repository. You can find them here. These repositories are where you can learn more about an image, what makes up the image, open GitHub Issues, and contribute PRs. If you’re having an issue with an image, especially if it’s a migration issue, you can open up a GitHub Issue and ask questions. You can always open up a Support ticket or reach out on CircleCI Discuss. 
```

Comment:
I think we may need a link to the image GH repository where "here" is stated. Also, should GitHub Issues be capitalized? Just a minor nit on that one. Finally, should we link to how to open a support ticket or post in Discuss?

Reply:
```suggestion
Every image has its own GitHub repository. You can find them here. These repositories are where you can learn more about an image, what makes up the image, open GitHub Issues, and contribute PRs. If you are having an issue with an image, especially if it’s a migration issue, you may open up a GitHub Issue and ask questions. You also may open a Support ticket or reach out on CircleCI Discuss. 
```

Reply:
GitHub do not capitalise "issues" in their docs so we shouldn't.

Reply:
This is something I do to differentiat when I as talking about an "issue" with software or something versus a GitHub "issue", the feature on their website. I changed it to lowercase.